setup.py

```diff
@@ -60,7 +60,7 @@
           To access, use the following in your analysis code:
           from csu_radartools import (
               csu_fhc, csu_kdp, csu_dsd, csu_liquid_ice_mass, csu_misc,
-              csu_blended_rain)
+              csu_blended_rain, fundamentals, common)
```

Comment:
Again, feel that common should be an internal import only, not externally advertised.